presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -245,4 +293,28 @@ else if (nullEnd) {
             return s -> level.name().equals(s);
         }
     }
+
```

Comment:
Well, there's a few methods on `Response` that make it easier to create responses `ok`, `accepted`, and `serverError` to name a few.

I've made a utility class with methods for three of the most common errors. I left out methods for the success codes, because those are used less frequently (about once per API function), and the main purpose of the class is to make code look nicer by saving space.